Formula/yosys.rb

```diff
@@ -22,6 +22,7 @@ class Yosys < Formula
   depends_on "readline"
 
   uses_from_macos "flex"
+  uses_from_macos "tcl-tk"
```

Comment:
We should maybe check if these are also needed at runtime.

Reply:
TCL is linked and will be required unless we disable the feature.

Don't know about `flex`.